dotnet/README.md

```diff
@@ -1,3 +1,6 @@
 [![Build Status](https://secure.travis-ci.org/cucumber/gherkin-dotnet.svg)](http://travis-ci.org/cucumber/gherkin-dotnet)
+[![Build status](https://ci.appveyor.com/api/projects/status/ka74of1ptpdfr6uv?svg=true)](https://ci.appveyor.com/project/SabotageAndi/gherkin)
```

Comment:
There is a [cucumberbdd](https://github.com/cucumber/aruba/issues/287#issuecomment-121805433) AppVeyor account created for and used by aruba (unless you didn't know).